Lib/distutils/command/bdist_wininst.py

```diff
@@ -4,6 +4,7 @@
 exe-program."""
 
 import sys, os
+import platform
```

Comment:
You can now removed this import which became unused.

Reply:
good catch

Reply:
done